process/OpenHW-Specification-Process.md

```diff
@@ -0,0 +1,180 @@
+| 2.3		| Creative Commons Attribution Share Alike 4.0 International | No patent grant included													| 
+| 3			| Essential patents must be disclosed by contributor | No, Contributors are not required to disclose essential patents 			| 
+| 4			| Copyright Grant included in contribution license:  | 																			|
+| 4.1		| Apache 2.0								| Yes, copyright grant included in contributions 									|
+| 4.2		| Solderpad 2.1								| Yes, copyright grant included in contributions 									|
+| 4.3		| Creative Commons Attribution Share Alike 4.0 International | Yes, copyright grant included	in contributions							| 
+| 5			| Contribution Process 						| Follows Eclipse Development Process (EDP) / Committers merging pull requests	/ Contributors sign Eclipse Contributor Agreement (ECA) or Member Committer and Contributor Agreement (MCCA)		| 			 
+
+
+#### Publication of the Specification
+
+| Number 	| Issue 									| Comment                                               							|
+| :---		| :------ 									| :-------------------------------------------------------------------------------- |
+| 1 		| Specification published by				| OpenHW Group on Github and/or ReadtheDocs and/or website								|
+| 2 		| Copyright holder on specification			| OpenHW Group on behalf of contributors (members), who hold the copyright jointly. 		|
+| 3			| License used for publication 				| Same license as used for contributions 											|
+| 4			| Copyright Grant included in specification publication 	 | As per publication license:										|	
+| 5.1		| Apache 2.0								| Yes, copyright grant included with OpenHW Group publication 							|
+| 5.2		| Solderpad 2.1								| Yes, copyright grant included with OpenHW Group publication 							|
+| 5.3		| Creative Commons Attribution Share Alike 4.0 International license | Yes, copyright grant included with OpenHW Group publication 			|								
+| 6			| Patent Grant included in specification publication 	|	As per publication license											|
+| 6.1		| Apache 2.0								| No patent grant provided by OpenHW but patents rights granted under contribution	|
+| 6.2		| Solderpad 2.1								| No patent grant provided by OpenHW but patents rights granted under contribution	|
+| 6.3		| Commons Attribution Share Alike 4.0 International license | No patent grant included with OpenHW publication					|								
+| 7			| Essential patents disclosed by OpenHW		| No																				|
+| 8 		| License text included within the specification text |  No - referral only 													|
+| 9			| "No-warranty" or disclaimer				| Required																|
+
+
+### Specification Revision Numbers
+
+OpenHW Group Specifications shall use semantic versioning https://semver.org/ with the Revision in the form X.Y.Z.
+The Revision number is combined with the Specification State (below), for example "1.0.0 - Released" to indicate both the Revision number and State.
```

Comment:
not always possible. Better to have postfixes (see semver.org) like -rc.1, -dev etc.